src/common/engine.js

```diff
@@ -121,6 +121,29 @@ class CurrencyEngine {
     )
   }
 
+  isSpendTx (edgeTransaction: EdgeTransaction): boolean {
+    if (edgeTransaction.nativeAmount) {
+      if (edgeTransaction.nativeAmount.slice(0, 1) === '-') {
+        return true
+      }
+      if (bns.gt(edgeTransaction.nativeAmount, '0')) {
+        return false
+      }
+    }
+    let out = true
```

Comment:
The `out` variable isn't really needed if you put `return false` inside the loop, and then `return true` at the end of the method. Still, no biggie - it obviously works fine.